[New] Configuring Nextcloud to use Linode Object Storage as an External Storage Mount

leslitagordita opened the pull request:
* *Note:* This guide relies on the Nextcloud One-Click Apps guide.

nmelehan commented:
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit c2e0b6fb4aebba93085cefd6b7e9854275a56169

https://deploy-preview-3411--nostalgic-ptolemy-b01ab8.netlify.app

nmelehan commented:
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit dc0a23a7468ade7bbe92fd9eda956e9aabbe832b

https://deploy-preview-3411--nostalgic-ptolemy-b01ab8.netlify.app

leslitagordita commented:
@andystevensname I just merged in Nextcloud OCA, so updated this guide with the proper link to the OCA guide. I added Singapore as an Object Storage region, since that was recently released, and I rebased the branch to bring in the recent menu update in our theme.
